ArtOfIllusion/src/artofillusion/math/Vec3.java

```diff
@@ -178,6 +178,20 @@ public final void normalize()
         z /= len;
       }
   }
+  
+  /** Crate a copy of this vector, scaled to the length of 1. If the length of this vector is 0, 
```

Comment:
Well, many Rust fans want you to think that. They may well eventually replace chunks of the system libraries that everyone depends upon, but there will probably still be a place for languages like Java. Future JVMs, though, might be written at least partly in Rust....